### PR TITLE
create carousel component - galleries of photos

### PR DESCRIPTION
**Pull request by Joanartdacosta**
create carousel component and css file (autoplay galleries)
install react slick
correction of photo's urls

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,17 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="./images/general/newlogo.png" />
+    <link
```
Comment (doublethink13) [inline public/index.html:6]: isn't there a way of bundling this css with the app?

https://react-slick.neostack.com/docs/get-started

seems you can 

```bash
npm install react-slick slick-carousel
```

and the import the css in your global css file

```css
import "~slick-carousel/slick/slick.css"; 
import "~slick-carousel/slick/slick-theme.css";
```

you shouldn't need to include the css as links

File: src/components/carousel/SimpleCarousel.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Slider from "react-slick";
+
+export default function SimpleCarousel(props) {
```
Comment (doublethink13) [inline src/components/carousel/SimpleCarousel.js:4]: this is almost the same code as the Carousel component

you should pass the settings as a props

then you only need one component

File: public/index.html (hunk 1 above)
Comment (Joanartdacosta) [reply public/index.html]: thank you for the observation! done

File: src/components/carousel/SimpleCarousel.js (hunk 2 above)
Comment (Joanartdacosta) [reply src/components/carousel/SimpleCarousel.js]: you are right!
nice comment, done

File: src/styles/index.css [hunk 3]
```diff
@@ -5,6 +5,10 @@
 @import "../components/navbar/navbar.css";
 @import "../components/footer/footer.css";
 @import "../components/social/social.css";
+@import "../components/carousel/carousel.css";
+/* react-slick-carousel */
```
Comment (doublethink13) [inline src/styles/index.css:9]: comment not needed

File: src/pages/Projects/ProjectsGallery.jsx [hunk 4]
```diff
@@ -1,29 +1,41 @@
...
-      {chalets.map((chalet) => createCard(chalet))}
-      <p>Estrada Fora - Travel Blog</p>
-      {roads.map((road) => createCard(road))}
-      <p>Mamaste - Lactation Consultations</p>
-      {mamastes.map((mamaste) => createCard(mamaste))}
-      <p>Reunion - Le Wagon Bootcamp's final project</p>
-      {reunions.map((reunion) => createCard(reunion))}
+      <div>
+        <p>
+          <a href="https://www.instagram.com/chalet_designinteriores/">
```
Comment (doublethink13) [inline src/pages/Projects/ProjectsGallery.jsx:23]: why is this not in s3?

File: src/styles/index.css (hunk 3 above)
Comment (Joanartdacosta) [reply src/styles/index.css:9]: there are another comments in this file to organize the index.css (components section, pages, fonts, etc.) 
In this case, I added the comment to understand that this import was related to react-slick configuration. It is not a component or the page's style

File: src/pages/Projects/ProjectsGallery.jsx (hunk 4 above)
Comment (Joanartdacosta) [reply src/pages/Projects/ProjectsGallery.jsx:23]: This is not an image, it is a link to instagram page

File: src/styles/index.css (hunk 3 above)
Comment (doublethink13) [reply src/styles/index.css:9]: you don't need comments for that

the import itself is enough information

at most, separate imports logically with new lines

some thing like

```css
@import "../components/footer/footer.css";
@import "../components/social/social.css";

@import "~slick-carousel/slick/slick.css";
@import "~slick-carousel/slick/slick-theme.css";
```

Comment (Joanartdacosta) [reply src/styles/index.css]: Thank you, the document was updated accordingly.

**Pull request merged**